src/status_im/contact/core.cljs

```diff
@@ -74,7 +75,12 @@
       (fx/merge cofx
                 {:db (assoc-in db [:contacts/new-identity] "")}
                 (upsert-contact contact)
-                (send-contact-request contact)))))
+                (mailserver/upsert-mailserver-topic
```

Comment:
how is this related to the message gaps?

Reply:
we need to add `public-key` to the set of `chat-ids` of `discovery`'s `mailserver-topic` in order to track requests per chat

Reply:
okay, cool, was curious, because it looks a bit weird... maybe a comment here will help, but I'll resolve this comment chain.